src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/AttributesAccordion.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useCallback, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { RootState } from 'src/store';
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemPanel,
+  AccordionItemButton,
+  AccordionItemPermanentBlock
+} from 'src/shared/accordion';
+
+import { getAttributesAccordionExpandedPanel } from './state/attributesAccordionSelector';
+import {
+  setAttributesAccordionExpandedPanel,
+  fetchAttributes
+} from './state/attributesAccordionActions';
+
+import {
+  Genes,
+  Transcripts,
+  Variations,
+  Location,
+  Orthologues,
+  Phenotypes,
+  Paralogues
+} from './sections';
+
+import styles from './AttributesAccordion.scss';
+
+type Props = StateProps & DispatchProps;
+
+const Attributes = (props: Props) => {
+  useEffect(() => {
+    props.fetchAttributes();
+  }, []);
```

Comment:
1) for default exports, can we name components the same way the file is named?
2) Is fetching attributes the responsibility of the attributes accordion or of the content section of custom download?

Reply:
1. That's done.

2. I initially had it in a way that content fetches the attributes, but then I thought it would be better to move it to the place where it is mainly used.